cpp/ps0001/ps0001solution.cpp

```diff
@@ -1,157 +1,25 @@
-    float num3;
+    unsigned short num3;
     cin >> num3;
 
-    cout << num1 << endl << num2 << endl << num3 << endl << "\n---------------\n" << num1 + num2 + num3;
-    return 1;
+    // outputting the result for summation process 3/3
+    cout << "---------------\n" << num1 + num2 + num3 << endl;
+    return 0;
 }
-
-
-
-//
-//int main()
-//{
-//    //عدد سكان عمارة
-//    short buildingResidents = 40;
-//
-//    //عدد البدايل لدوا في الصيدلية
-//    short medicinAlternatives = 4;
-//
-//    //أفراد تيم في شركة برمجة
-//    short teamMembers = 5;
-//
-//    //سعر واحدة من مخبوزات الفرن
-//    float bakedPrice = 10.5;
-//
-//    //الحرف الأول من اسم شخص
-//    char personFirstLetter = 'h';
-//
-//    //عدد المواليد في دولة تعدادها كبير زي الصين
-//    long populationNumber = 2912534765;
-//
-//    //ناتج اكثر دقة لقسمة رقم على 7
-//    double result = 83.8571429;
-//
-//    //فاتورة الكهربا لخط انتاج في مصر
-//    double electricityBill = 332188737363636.64564546;
-//
-//    //عمر أشخاص كبار و صغار ف السن بالثانية
-//    long ageInSeconds = 420480000;
-//
-//    return 1;
-//}
-
-
-
-
-
-//
-//int main()
-//{
-//
-//
-//
-//
-//
-//
-//
-//
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//   //cout << " *********************************\n";
-//
-//
-//    //cout << "*     *\n";
-//    //cout << "*     *\n";
-//    //cout << "* * * *\n";
-//    //cout << "*     *\n";
-//    //cout << "*     *\n";
-//    ////////////////////////////////////
-//  /* cout << " *********************************\n";
-//   cout << " name: Hajar saeed AbduAziz\n";
-//   cout <<" age:39 years\n";
-//   cout << " city : Ciro\n";
-//   cout << " country: Egypt\n";
-//   cout << " *********************************\n";
-//
-//   cout << "I Love Programming \n\nI promise to be the best develober ever !\n\n";
-//   
-//   cout << "I know it will take sometime to practice, but i will achieve my goal \n\n";
-//   cout << "Best regards\n Hajar saeed makhlof";*/
-//  
-//
-//    /*
-//   
-//    cout << "please enter rectangle length ,  width: \n";
-//    int width , length ;
-//   
-//  
-//    cin >> length >> width ;
-//
-//   
-//
-//    cout << "the area of rectangle with length  "<<length << " and width "<< width << " = " << length * width;
-//
-//    */
-//
-// 
-//
-//    //firstQuestion();
-//   // secondQuestion();
-//  //  question7();
-//   // question9();
-//   // question10();
-//   // question14();
-//   // question15();
-//   // question17();
-//   // question19();
-//   // question20();
-//    //question21();
-//   // question22();
-//   // question31();
-//    //question35();
-//   // question39();
-//   // question1();
-//   // question40();
-//    //question42();
-//  //question43();
-//    // question47();
-//    // question48();
-//
-//
-//
-//   
-//   
-// 
-//
-//   
-//
-//
-//}
-
+// good work ✌️ 7/10 

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [ ] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code